tests/Filter/FilterTestCase.php

```diff
@@ -18,17 +18,43 @@
 use Doctrine\ORM\Query\Expr\Orx;
 use Doctrine\ORM\QueryBuilder;
 use PHPUnit\Framework\TestCase;
+use Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQuery;
 
-class FilterTestCase extends TestCase
+class TestQueryBuilder extends QueryBuilder
```

Comment:
IMO, we should declare this fixture after the PSR-4 class.

Reply:
Done